docs/content/Authentication-&-Authorization.md

```diff
@@ -5,23 +5,50 @@ category: Authentication & Authorization
 menuOrder: 1
 ---
 
+In Cube.js, access control or authorization is based on the **security context**. The diagram below shows how it works during the request processing in Cube.js:
+
+
+<p
+  style="text-align: center"
+>
+  <img
+  src="https://raw.githubusercontent.com/statsbotco/cube.js/master/docs/content/authentication-overview.png"
+  style="border: none"
+  width="80%"
+  />
+</p>
+
+Authentication is handled outside of Cube.js. Your auth server issues JWTs to your client app, which, when sent as part of the request, are verified and decoded by the Cube.js to get security context claims to evaluate access control rules.
```

Comment:
```suggestion
Authentication is handled outside of Cube.js. Your authentication server issues JWTs to your client application, which, when sent as a part of the request, are verified and decoded by Cube.js to get security context claims and evaluate access control rules.
```

I suggest that we never use such contractions as "auth" (confusing) and "app" (unprofessional) in our docs.